SmartTutor/Startup.cs

```diff
@@ -66,7 +66,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IWorkspaceCreator, NoWorkspaceCreator>();
             services.AddScoped<ILearnerRepository, LearnerDatabaseRepository>();
 
-            services.AddScoped<IInstructor, VARKRecommender>();
+            services.AddScoped<VARKRecommender, VARKRecommender>();
```

Comment:
This article should help with the composite IInstructor https://andrewlock.net/adding-decorated-classes-to-the-asp.net-core-di-container-using-scrutor/ - make note of it, as it will serve as an interesting segment in your BSc.